docs/source/integrations.mdx

```diff
@@ -21,13 +21,41 @@ quantization_config = BitsAndBytesConfig(load_in_4bit=True, bnb_4bit_compute_dty
 # PEFT
 With `PEFT`, you can use QLoRA out of the box with `LoraConfig` and a 4-bit base model.
 
-Please review the [bitsandbytes section in the Accelerate docs](https://huggingface.co/docs/peft/developer_guides/quantization#quantize-a-model).
+Please review the [`bitsandbytes` section in the Accelerate docs](https://huggingface.co/docs/peft/developer_guides/quantization#quantize-a-model).
```

Comment:
```suggestion
Please review the [`bitsandbytes` section in the PEFT docs](https://huggingface.co/docs/peft/developer_guides/quantization#quantize-a-model).
```

Reply:
ah, I noticed that too, but fixed it in `main` so that the correction is live immediately..